doc/source/reference.rst

```diff
@@ -107,14 +107,20 @@ charts.
 
 However, the OpenStack-Helm project has a repository, `openstack-helm-images`,
 containing a reference implementation for the images. That repository
-holds the images used for the OpenStack-Helm project charts.
-
-These images are built with Docker.
-`openstack-helm-images` provides Dockerfiles for all the non-OpenStack images.
-For the OpenStack images, `openstack-helm-images` only has a shell script, a
-thin wrapper around `LOCI`. `LOCI` is the official OpenStack project to
-build OCI compliant images of OpenStack. It follows a source based deployment
-model.
+holds the images used for the OpenStack-Helm project charts. All its images
+are built with Docker.
+
+`openstack-helm-images` provides Dockerfiles directly for all the
+non-OpenStack images.
+
+For the OpenStack images, `openstack-helm-images` contains shell scripts,
+situated in `openstack/loci/`. The `build.sh` script is a thin wrapper around
+`LOCI`. `LOCI` is the official OpenStack project to build OCI compliant
+images of OpenStack projects. It uses `docker build` to construct images from
+openstack sources and their requirements are expressed in `bindep` files.
+The `build.sh` runs `LOCI` for the master branch. Other branches can be built
+suing `build-{branchname}.sh` where `branchname` is the name of the OpenStack
+branch (for example, `queens`).
```

Comment:
Can you mention the ansible variable that user could change to build for different branch? (`loci_build_command`)

Reply:
Will update.